src/components/Reports/SalaryCalculator/Landing/NewSalaryCalculationLanding/SalaryOverviewCard.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {
+  Card,
+  CardContent,
+  CardHeader,
+  Grid,
+  Stack,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { useTranslation } from 'react-i18next';
+import { useLandingData } from '../useLandingData';
+
+const StyledCard = styled(Card)(({ theme }) => ({
+  marginTop: theme.spacing(3),
+  marginBottom: theme.spacing(3),
+}));
+
+const StyledValueTypography = styled(Typography)(({ theme }) => ({
+  fontWeight: 'bold',
+  color: theme.palette.primary.main,
+  fontSize: theme.typography.h3.fontSize,
+}));
+
+export const SalaryOverviewCard: React.FC = () => {
+  const { t } = useTranslation();
+  const theme = useTheme();
+  const {
+    name,
+    accountBalance,
+    staffAccountId,
+    salaryData: { currentGrossSalary },
+  } = useLandingData();
+
+  return (
+    <StyledCard>
+      <CardHeader
+        title={<Typography variant="h5">{name}</Typography>}
+        subheader={
+          <Typography variant="body1" color="textSecondary">
+            {t(`${staffAccountId}`)}
```

Comment:
The first argument to `t` calls always has to be a string literal. In this case, we don't need to localize the staff account id because it's just a number.

```suggestion
            {staffAccountId}
```